spec/aep-63/README.md

```diff
@@ -0,0 +1,131 @@
+Progress is tracked in the [Managed Wallets API Milestone](https://github.com/akash-network/console/milestone/10)
+
+### API Endpoints
+The following API endpoints have been identified at the time of writing this spec. As we work on onboaring customers to use the API we will either update this spec or create new roadmap items to add additional API endpoints. 
+
+Note that while there are a lot of API endpoints listed here - the managed wallets user will only need to exercise a subset of them. The rest are listed here in the interest of full specification.
+
+#### API Authentication & Management
+
+While these endpoints will be made available to the user, we anticipate that most users will use the UI to manage their keys
+
+`POST /v1/users/api-keys` (create)
+`GET /v1/users/api-keys` (read/ list all keys)
+`GET /v1/users/api-keys/{id}` (read/ list specific API key details)
+`PATCH /v1/users/api-keys/{id}` (update)
+`DELETE /v1/users/api-keys/{id}` (delete)
+
+In addition there needs to be validation of the API keys at the middleware level so that it applies to any user specific endpoint
+
+Implementation/ status tracked in [Console issue #768](https://github.com/akash-network/console/issues/768)
+
+#### Certificate Management
+
+While there will be a documented API for this - it will likely be handled automagically under the hood for the managed wallet user. If it is handled for the user we may not implement the list and revoke endpoints
+
+`POST /v1/certificates` (create cert)
+`GET /v1/certificates` (list all certs)
+`DELETE /v1/certificates/{id}` (revoke cert)
+
+#### Deployment Creation
+
+`POST /v1/deployments` (create deployment based on `SDL/ YAML` and escrow `deposit`)
+
+Implementation tracked in [issue #767](https://github.com/akash-network/console/issues/767)
+
+#### Bid Selection
+
+`GET v1/bids/{dseq}` (get bids for a `dseq`)
+
+Implementation tracked in [issue #767](https://github.com/akash-network/console/issues/767)
+
+#### Lease Creation
+
+`POST /v1/leases` (create lease based on a payload that is an array of sdl, dseq, gseq, provider combinations)
+
+Note that Akash supports creating multiple leases from a single SDL - where each lease can be for a different service (with a different container image) and deployed to a different provider
+
+Implementation tracked in [issue #767](https://github.com/akash-network/console/issues/767)
+
+#### Listing Deployments
+
+`GET /v1/deployments` (list all deployments)
+`GET /v1/deployments/{dseq}` (list deployment detail for a specific deployment)
+
+Implementation tracked in [issue #767](https://github.com/akash-network/console/issues/767)
+
+#### Deployment Closure
+
+`DELETE /v1/deployments/{dseq}` (close a specific deployment)
+
+Implementation tracked in [issue #767](https://github.com/akash-network/console/issues/767)
+
+#### Funding Deployments
+
+For viewing or adding funds to the escrow of a specific deployment
+
+`GET /v1/deployments/deposit{dseq}` (retrieve escrow balance details)
```

Comment:
```suggestion
`GET /v1/deployments/deposit/{dseq}` (retrieve escrow balance details)
```

Reply:
I don't think we need this endpoint as the get deployment details will include the escrow balance

Reply:
ok, will replace